src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,23 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import mate.academy.dao.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
 public class Main {
-    public static void main(String[] args) {
+    private static final Injector injector = Injector.getInstance("mate.academy");
 
+    public static void main(String[] args) {
+        Book book = new Book();
+        book.setPrice(BigDecimal.valueOf(207));
+        book.setTitle("Sherlock Holmes");
+        book.setId(4L);
+        BookDao bookDao = (BookDao) injector.getInstance(BookDao.class);
+        bookDao.create(book);
+        bookDao.findById(4L);
+        bookDao.deleteById(3L);
+        bookDao.update(book);
+        bookDao.findAll();
```

Comment:
Consider adding some print statements between some methods to check if results are correct.